doc/man1/ciphers.pod

```diff
@@ -156,19 +156,20 @@ The cipher string B<@SECLEVEL=n> can be used at any point to set the security
 level to B<n>, which should be a number between zero and five, inclusive.
 See L<SSL_CTX_set_security_level> for a description of what each level means.
 
+The cipher list can be prefixed with B<DEFAULT> keyword, which enables
+the default cipher list as defined below.  Unlike the cipher string,
```

Comment:
s/Unlike the cipher string/Unlike other cipher strings/

Reply:
Wouldn't that imply `DEFAULT` is a cipher string?  Because it's not... (at least that's what I'm trying to say)